### DISCUSSION TITLE
Please add GPEN-BFR-1024 and GPEN-BFR-2048 Face Restoration Models !

### DISCUSSION THREAD

**Discussion by web2299** [vladmandic/sdnext#3023] [Ideas]
https://github.com/yangxy/GPEN

Recently i tried **GPEN-BFR-1024** and **GPEN-BFR-2048** in FaceFusion and they looks so much better then Codeformer and Gfpgan_1.4 !
So can you please implement them in Automatic too !? Thank you very much in advance !

**Comment by brknsoul** [comment]
Have you tried vlad's Face Restore under the Advanced accordion?
Also, it's an extension available here https://github.com/diffus-me/sd-webui-facefusion

**Reply by web2299** [reply]
No i havent tried - i will check it - thanks !
